src/procedures/procedure.h

```diff
@@ -43,7 +42,13 @@ const char *Procedure_GetOutput(const ProcedureCtx *proc, uint output_idx);
 bool Procedure_ContainsOutput(const ProcedureCtx *proc, const char *output);
 
 /* Returns true if procedure is read-only. */
-bool Proc_ReadOnly(const char *proc_name);
+bool Procedure_ReadOnly(const char *proc_name);
```

Comment:
Now that we have `Procedure_IsReadOnly` do we really need both `Procedure_IsReadOnly` and `Procedure_ReadOnly` can we do with just one?